dev/devicelab/manifest.yaml

```diff
@@ -283,6 +283,24 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
 
+  flutter_create_offline_test_linux:
+    description: >
+      Tests the `flutter create --offline` command.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```

Comment:
All new devicelab tasks should be marked `flaky: true` for a period of time, just in case